tests/base.py

```diff
@@ -129,6 +129,9 @@ def record_to_bk_value(self, stream, record):
     def tap_name(self):
         return "tap-xero"
 
+    def name(self):
+        return "tap_tester_xero_connection"
```

Comment:
This change looks out of scope of current issue we are trying to address but if it is required please use the same name as before.

```suggestion
        return "tap_tester_xero_common_connection"
```

Reply:
Done